openstack/networking/v2/extensions/vpnaas/ikepolicies/doc.go

```diff
@@ -17,5 +17,12 @@ Example to Create an IKE policy
 		panic(err)
 	}
 
+Example to Show the details of a specific IKE policy by ID
+
+	policy, err := ikepolicies.Get(client, "f2b08c1e-aa81-4668-8ae1-1401bcb0576c").Extract()
+    if err != nil {
```

Comment:
Small nit to fix the indenting here. I'm going to go ahead and merge this since everything else is good, but could you fix this in a future PR?

This inline documentation will be displayed as the official documentation at godoc.org, so we want to make sure it's correct.